manager/controllers/app/blueprint_controller.go

```diff
@@ -85,7 +89,7 @@ func (r *BlueprintReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 		}
 	}
 
-	log.Info("blueprint reconcile cycle completed", "result", result)
+	log.Trace().Msg("blueprint reconcile cycle completed.") // TODO - Add result to log?
```

Comment:
It is a debug printout. Any way to indicate this?

Reply:
Would you prefer this have verbosity level debug rather than trace?

Reply:
Yes, debug instead of trace.

Reply:
Will change